### ISSUE TITLE
Letsencrypt config prevents renewal

### ISSUE THREAD

**Issue by TacoScheltema** [modoboa/modoboa#1280]
# Impacted versions

* Modoboa: 1.9.1 (and previous versions)
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce
Use the installer to setup Modoboa and enable Letsencrypt

# Current behavior
Certificate renewal fails

# Expected behavior
Certificate is renewed in time

After installation the Letsencrypt authenticator is set to 'standalone' in /etc/letsencrypt/renewal/<domain>.conf like so:

```
[renewalparams]
authenticator = standalone
```

This will cause the renewal to fail due to port 443 being in use.
by changing the authenticator to 'nginx' after installation has completed the renewal process will then use nginx for the renewal and succeed.

```
[renewalparams]
authenticator = nginx
```

**Comment by tonioo** [comment]
@TacoScheltema Does it require a modification on nginx side?

**Comment by TacoScheltema** [comment]
@tonioo no, all it will do is use nginx to verify the certificate renewal.
With the authenticator set to standalone, certbot will try to start it's own listener on port 443 and fail since nginx is already listening on that port. 
There's no change needed to nginx configuration.

**Comment by tonioo** [comment]
Thank you for the feedback, I've fixed the installer (https://github.com/modoboa/modoboa-installer/issues/165).

**Comment by TacoScheltema** [comment]
Awesome! thanks 👍
